spec/takeout_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'takeout'
+
+describe 'Takeout' do
+  let(:takeout){Takeout.new}
+  let(:menu){ double('menu', menu: food) }
+
+  xit 'can place an order' do
+
+  expect(takeout.confirm())
```

Comment:
Style/MethodCallParentheses: Do not use parentheses for method calls with no arguments.